topics/statistics/tutorials/classification_machinelearning/workflows/ml_classification.ga

```diff
@@ -0,0 +1 @@
c-41a4-b53b-c04533b16742", "errors": null, "name": "Plot confusion matrix, precision, recall and ROC and AUC curves", "post_job_actions": {}, "label": null, "inputs": [], "position": {"top": 370, "left": 670}, "annotation": "", "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/plotly_ml_performance_plots/plotly_ml_performance_plots/0.2", "type": "tool"}, "19": {"tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/sklearn_ensemble/sklearn_ensemble/1.0.8.2", "tool_version": "1.0.8.2", "outputs": [{"type": "tabular", "name": "outfile_predict"}], "workflow_outputs": [], "input_connections": {"selected_tasks|infile_data": {"output_name": "output", "id": 2}, "selected_tasks|infile_model": {"output_name": "outfile_object", "id": 14}}, "tool_state": "{\"__page__\": null, \"selected_tasks\": \"{\\\"__current_case__\\\": 0, \\\"header\\\": \\\"true\\\", \\\"infile_data\\\": null, \\\"infile_model\\\": null, \\\"prediction_options\\\": {\\\"__current_case__\\\": 0, \\\"prediction_option\\\": \\\"predict\\\"}, \\\"selected_task\\\": \\\"load\\\"}\", \"chromInfo\": \"\\\"/opt/galaxy/tool-data/shared/ucsc/chrom/?.len\\\"\", \"__rerun_remap_job_id__\": null}", "id": 19, "tool_shed_repository": {"owner": "bgruening", "changeset_revision": "3ab7af14f1b5", "name": "sklearn_ensemble", "tool_shed": "toolshed.g2.bx.psu.edu"}, "uuid": "aa1bed69-0ee6-4ade-b9b2-6d2ce89bb36e", "errors": null, "name": "Ensemble methods", "post_job_actions": {}, "label": null, "inputs": [], "position": {"top": 490, "left": 670}, "annotation": "", "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/sklearn_ensemble/sklearn_ensemble/1.0.8.2", "type": "tool"}, "20": {"tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/plotly_ml_performance_plots/plotly_ml_performance_plots/0.2", "tool_version": "0.2", "outputs": [{"type": "html", "name": "output_confusion"}, {"type": "html", "name": "output_prf"}, {"type": "html", "name": "output_roc"}], "workflow_outputs": [], "input_connections": {"infile_trained_model": {"output_name": "outfile_object", "id": 14}, "infile_output": {"output_name": "outfile_predict", "id": 19}, "infile_input": {"output_name": "output", "id": 1}}, "tool_state": "{\"__page__\": null, \"infile_trained_model\": \"null\", \"__rerun_remap_job_id__\": null, \"infile_output\": \"null\", \"infile_input\": \"null\", \"chromInfo\": \"\\\"/opt/galaxy/tool-data/shared/ucsc/chrom/?.len\\\"\"}", "id": 20, "tool_shed_repository": {"owner": "bgruening", "changeset_revision": "62e3a4e8c54c", "name": "plotly_ml_performance_plots", "tool_shed": "toolshed.g2.bx.psu.edu"}, "uuid": "635ba9ca-1f96-44ff-970e-d2877b12292c", "errors": null, "name": "Plot confusion matrix, precision, recall and ROC and AUC curves", "post_job_actions": {}, "label": null, "inputs": [], "position": {"top": 10, "left": 890}, "annotation": "", "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/plotly_ml_performance_plots/plotly_ml_performance_plots/0.2", "type": "tool"}}, "annotation": "", "a_galaxy_workflow": "true"}
```

Comment:
please reformat this file https://github.com/usegalaxy-eu/workflow-testing#pretty-printing-worfklow-json

and add a `tags`, as well as an `annotation`